.github/renovate.json5

```diff
@@ -141,19 +141,21 @@
     {
       // Disable app minor version update in the previous Camunda Helm charts.
       "enabled": false,
-      "matchPackageNames": ["/.*camunda.*/"],
+      "matchPackageNames": ["/.*camunda\/.*/"],
       "matchFileNames": [
-        "charts/camunda-platform-8*/values.yaml",
-        "charts/camunda-platform-8*/values-latest.yaml"
+        "charts/camunda-platform-8.*/values.yaml",
+        "charts/camunda-platform-8.*/values-latest.yaml",
+        "charts/camunda-platform-latest/values.yaml",
+        "charts/camunda-platform-latest/values-latest.yaml"
       ],
       "matchUpdateTypes": ["minor"]
     },
     {
       // Enable app patch version update in the previous Camunda Helm charts.
       "matchDatasources": ["helmv3", "helm-values", "docker", "regex"],
       "matchPaths": [
```

Comment:
While we are on it, I just noticed that `matchPaths` has been renamed, and it doesn't show in the [renovatebot options](https://docs.renovatebot.com/configuration-options/).

Reply:
I tested that matchFileNames works, and it does.